tests/RuleSet/RuleSetTest.php

```diff
@@ -406,6 +407,25 @@ public static function providePhpUnitTargetVersionHasSetCases()
         }
     }
 
+    private function sortNestedArray(array $array)
+    {
+        foreach ($array as $key => $element) {
+            if (!\is_array($element)) {
+                continue;
+            }
+            $array[$key] = $this->sortNestedArray($element);
+        }
+
+        // sort by key if associative, by values otherwise
```

Comment:
what about configs like `ordered_class_elements => [order => $arrayOfItems]`? 
order of items in `arrayOfItems` is important and `[construct, destruct]` is different than `[destruct, construct]`

Reply:
I guess we can add an exception in future if needed.

Reply:
i gave example of what we already have in the codebase.
I'm afraid of maintaining the exceptions list would be tedious task that nobody would remember about